### PR TITLE
[tw] Localize `Scalability` for Traditional Chinese

### PR DESCRIPTION
**Pull request by win5923**
### Describe your changes

Origin content from https://glossary.cncf.io/scalability/

### Related issue number or link (ex: `resolves #issue-number`)

`resolves` #2491 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/scalability.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: 可擴展性
+status: Completed
+category: 屬性
+tags: ["基本原理", "屬性", ""]
+---
+
+可擴展性指的是一個系統能夠擴大成長的能力，
+也就是增加系統執行其預定功能的能力。
+舉例來說，
+[Kubernetes](/zh-tw/kubernetes/) [叢集]((/zh-tw/cluster/)) 透過增減 [容器化](/zh-tw/containerization/) 應用程式的數量來實現擴展，
```
Comment (johnlinp) [inline content/zh-tw/scalability.md:11]: 我個人覺得 "實現" 這個詞在台灣的語境比較像是 "實現夢想" 或是 "實現諾言" 這種用法，
在軟體上比較少用。

因為 "擴展" 已經是一個動詞了，所以我建議這邊直接用 "擴展" 這個動詞就好：

```
Kubernetes 叢集透過增減容器化應用程式的數量來擴展
```

原文:

> For example, a [Kubernetes](https://glossary.cncf.io/kubernetes/) [cluster](https://glossary.cncf.io/cluster/) scales by increasing or reducing the number of [containerized](https://glossary.cncf.io/containerization/) apps, but that scalability depends on several factors.

File: content/zh-tw/scalability.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+但這種可擴展性取決於幾個因素。
+它擁有多少 [節點](/zh-tw/nodes/)、每個節點可以處理多少 [容器](/zh-tw/container/)，
+以及控制平面能夠支援多少記錄和操作？
+
+可擴展性的系統使增加容量變得容易。
+主要區分為兩種擴展方法。一方面，
+有 [水平擴展]((/zh-tw/horizontal-scaling/)) 增加更多節點以處理增加的負載。
+相比之下，
+在 [垂直擴展](/zh-tw/vertical-scaling/) 中，
```
Comment (johnlinp) [inline content/zh-tw/scalability.md:20]: nit: 我注意到你在中文專有名詞的左右有加上空白：

image

但我看我們目前的翻譯中，似乎沒有把中文專有名詞的左右加上空白的習慣，比方說 "分散式系統" ：

image

File: content/zh-tw/scalability.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+tags: ["基本原理", "屬性", ""]
+---
+
+可擴展性指的是一個系統能夠擴大成長的能力，
+也就是增加系統執行其預定功能的能力。
+舉例來說，
+[Kubernetes](/zh-tw/kubernetes/) [叢集]((/zh-tw/cluster/)) 透過增減 [容器化](/zh-tw/containerization/) 應用程式的數量來實現擴展，
+但這種可擴展性取決於幾個因素。
+它擁有多少 [節點](/zh-tw/nodes/)、每個節點可以處理多少 [容器](/zh-tw/container/)，
+以及控制平面能夠支援多少記錄和操作？
```
Comment (johnlinp) [inline content/zh-tw/scalability.md:14]: 在英文裡，我常看到多個問句是用逗點隔開的，但在中文裡，我比較少看到這樣的用法。
中文裡，我覺得比較常看到的是用問號隔開的，像是這樣：

```
它擁有多少節點？
每個節點可以處理多少容器？
以及控制平面能夠支援多少記錄和操作？
```

不知道你覺得呢？

原文：

> How many [nodes](https://glossary.cncf.io/nodes/) does it have, how many [containers](https://glossary.cncf.io/container/) can each node handle, and how many records and operations can the control plane support?

File: content/zh-tw/scalability.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+可擴展性的系統使增加容量變得容易。
+主要區分為兩種擴展方法。一方面，
```
Comment (johnlinp) [inline content/zh-tw/scalability.md:17]: 這邊我建議把 "On the one hand" 翻譯成 "其中一種方法" ，好像會比較通順一點。

原文:

> On the one hand, there is [horizontal scaling](https://glossary.cncf.io/horizontal-scaling/) which adds more nodes to handle increased load. In contrast, in [vertical scaling](https://glossary.cncf.io/vertical-scaling/) individual nodes are made more powerful to perform more transactions (e.g. by adding more memory or CPU to an individual machine).

File: content/zh-tw/scalability.md (hunk 2 above)
Comment (hwchiu) [reply content/zh-tw/scalability.md:20]: 我傾向中文不需要保留空白，英文有就好

File: content/zh-tw/scalability.md (hunk 3 above)
Comment (hwchiu) [reply content/zh-tw/scalability.md:14]: 我個人覺得還好，因為最後產出的結果還是放同一行，這邊應該只是為了跟英文版的內容對齊

Comment (johnlinp) [reply content/zh-tw/scalability.md:14]: @hwchiu 抱歉，我可能表達不清楚，
我的重點不是在分行，
我想講的是建議用問號隔開問句，
而不是用逗號，
謝謝。

File: content/zh-tw/scalability.md (hunk 4 above)
Comment (win5923) [reply content/zh-tw/scalability.md]: 修正完畢!

File: content/zh-tw/scalability.md (hunk 3 above)
Comment (hwchiu) [reply content/zh-tw/scalability.md:15]: @johnlinp 哦哦，懂了，感謝解釋
原文的問句的確是針對三個問題，所以中文這邊我也支持有三個問號來區別彼此

**Review by johnlinp — APPROVED** [review]
謝謝!

File: content/zh-tw/scalability.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+舉例來說，
+[Kubernetes](/zh-tw/kubernetes/) [叢集]((/zh-tw/cluster/))透過增減[容器化](/zh-tw/containerization/)應用程式的數量來擴展，
+但這種可擴展性取決於幾個因素。
+它擁有多少[節點](/zh-tw/nodes/)?
+每個節點可以處理多少[容器](/zh-tw/container/)?
+以及控制平面能夠支援多少記錄和操作？
+
+可擴展性的系統使增加容量變得容易。
+主要區分為兩種擴展方法。
+其中一種方法，
```
Comment (ydFu) [inline content/zh-tw/scalability.md:19]:
```suggestion
其中之一，
```

File: content/zh-tw/scalability.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+有[水平擴展](/zh-tw/horizontal-scaling/)增加更多節點以處理增加的負載。
```
Comment (ydFu) [inline content/zh-tw/scalability.md:20]:
```suggestion
[水平擴展](/zh-tw/horizontal-scaling/)添加更多節點以處理增加的負載。
```
en
```
On the one hand, there is [horizontal scaling](/horizontal-scaling/) which adds more nodes to handle increased load. 
```

單一句子有兩個重複的單字，不像中文書寫慣用方式

Comment (win5923) [reply content/zh-tw/scalability.md]: 修改完成!

File: content/zh-tw/scalability.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+status: Completed
+category: 屬性
+tags: ["基本原理", "屬性", ""]
+---
+
+可擴展性指的是一個系統能夠擴大成長的能力，
+也就是增加系統執行其預定功能的能力。
+舉例來說，
+[Kubernetes](/zh-tw/kubernetes/) [叢集]((/zh-tw/cluster/))透過增減[容器化](/zh-tw/containerization/)應用程式的數量來擴展，
+但這種可擴展性取決於幾個因素。
```
Comment (ydFu) [inline content/zh-tw/scalability.md:12]:
```suggestion
但這種可擴展性取決於幾項因素。
```

File: content/zh-tw/scalability.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+它擁有多少個[節點](/zh-tw/nodes/)？
+每個節點可以處理多少[容器](/zh-tw/container/)？
```
Comment (ydFu) [inline content/zh-tw/scalability.md:14]:
```suggestion
每個節點可以處理多少個[容器](/zh-tw/container/)？
```

**Review by ydFu — COMMENTED** [review]
The whole thing looks good. After modifying the quantifiers, I compressed multiple git commits into one. I can merge this PR. If you don’t know how to compress git commits, you can ask in slack.

**Comment by ydFu** [comment]
It seems that the change has been changed to other documents. Check the commit to be contributed.

image


https://github.com/cncf/glossary/compare/2e050009cfbd60c911f9e4ad740c429412bb11af..5e37df2ce54faaeca09839a989ff954c8ebb54f4

**Comment by win5923** [comment]
Sorry, I forgot to merge it.

**Review by hwchiu — CHANGES_REQUESTED** [review]
麻煩 rebase 一下變成 1 筆 commit，剩下沒問題就可以合併了

感謝

**Review by ydFu — APPROVED** [review]
LGTM

**Review by hwchiu — APPROVED** [review]
LGTM

**Pull request merged**